src/scss/_color.scss

```diff
@@ -56,6 +56,12 @@ $emc-color-map: (
         emc-neon: (background-color: #E1FF32, color: #000),
         emc-lime: (background-color: #C9FB36, color: #000),
         emc-olive: (background-color: #A0BD00, color: #000),
+
+        emc-link: (background-color: #fff, color: #0645AD),
+        emc-link-hover: (background-color: #fff, color: #005760),
+        emc-link-active: (background-color: #fff, color: #005760),
+        emc-link-focus: (background-color: #fff, color: #005760),
+        emc-link-visited: (background-color: #fff, color: #0B0080),
```

Comment:
The _colors.scss file should only reference colors.  Having "*-link-*" in the name seems to indicate that these reference do not belong here.  The colors in this file should already include all of the colors provided in the Figma document.  I would recommend removing these definitions and changing the references in the theme to point to whichever color is closest to the ones that you had defined here.

Reply:
I see what you mean. I will just use the blue ones for now then until Jes defines the color palette for these.